code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -180,11 +180,10 @@
 	// If we're about to display this group to a mob who's client is more recent than the last known version with working CENTER, then we need to remake the relays
 	// with the correct screen_loc using the relay override
 	if(viewing_hud.mymob?.client?.byond_build > MAX_CLIENT_BUILD_WITH_WORKING_SECONDARY_MAPS)
-		for(var/thing in plane_masters)
```

Comment:
My bad, I didn't read the code properly. Undo my changes and put back your code, for the for loop

Reply:
lmao oops